courses/views.py

```
@@ -0,0 +1,38 @@
+from django.shortcuts import render, get_object_or_404, redirect
+from django.contrib.auth.decorators import login_required
+from django.contrib import messages
+from .models import Course, Session, Enrollment
+
+def course_list(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `course_list`

(ANN201)